Assets/Scripts/PlayerMovement.cs

```diff
@@ -22,10 +24,17 @@ void FixedUpdate()
         Vector3 move = new Vector3(Input.GetAxis("Horizontal"), 0, Input.GetAxis("Vertical"));
         move = Vector3.ClampMagnitude(move, 1f);
         controller.Move(move * Time.fixedDeltaTime * playerSpeed * GetMovementCoef());
-
         if (move != Vector3.zero)
         {
-            gameObject.transform.forward = move;
+            var r = Quaternion.LookRotation(move);
+            transform.rotation = Quaternion.RotateTowards(transform.rotation, r, rotateSpeed * Time.fixedDeltaTime);
```

Comment:
please no variables with one character, like `r` here. I understand that it's some kind of rotation, but at least call it `rotation` or more specific